### PR TITLE
v4.0.0

### PR DESCRIPTION
**Pull request by ljharb**
I've added the "exports" field, but only with the "main".

If I'm missing any entry points that are needed, I can add them in this PR, or we can add them later in a semver-minor.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,8 +1,12 @@
 {
   "name": "eslint-find-rules",
-  "version": "3.6.1",
+  "version": "4.0.0",
   "description": "Find built-in ESLint rules you don't have in your custom config.",
   "main": "dist/lib/rule-finder.js",
+  "exports": {
+    ".": "dist/lib/rule-finder.js",
```
Comment (PaperStrike) [inline package.json:7]: https://nodejs.org/api/packages.html#exports

> All paths defined in the `"exports"` must be relative file URLs starting with `./`.

👀

**Comment by PaperStrike** [comment]
A reminder for the docs. 👀 It returns a promise now.
https://github.com/sarbbottam/eslint-find-rules/blob/master/README.md#as-a-required-module

**Comment by ljharb** [comment]
Thanks! Done in b530d24

File: package.json (hunk 1 above)
Comment (ljharb) [reply package.json:7]: oops, thanks.

**Review by ta2edchimp — APPROVED** [review]

**Pull request merged**